src/extensions/list_status.rs

```diff
@@ -171,9 +172,9 @@ mod tests {
                     * LIST (\\UnMarked) \".\" feeds\r\n\
                     * LIST () \".\" feeds.test\r\n\
                     * STATUS feeds.test (HIGHESTMODSEQ 757)\r\n";
-        let (mut send, recv) = mpsc::channel();
-        let fetches = ExtendedNames::parse(lines.to_vec(), &mut send).unwrap();
-        assert!(recv.try_recv().is_err());
+        let mut queue = VecDeque::new();
+        let fetches = ExtendedNames::parse(lines.to_vec(), &mut queue).unwrap();
+        assert!(queue.pop_front().is_none());
```

Comment:
```suggestion
        assert_eq!(queue.pop_front(), None);
```
So we see what's there if the test fails.